tools/esp8266/app.cpp

```diff
@@ -53,18 +53,22 @@ void app::loop(void) {
     mWebInst->loop();
 
     if(checkTicker(&mUptimeTicker, mUptimeInterval)) {
```

Comment:
this must be removed because the next line, we don't have to compare millis twice